common/src/main/kotlin/de/hpi/cloud/common/types/DateTime.kt

```diff
@@ -0,0 +1,98 @@
+package de.hpi.cloud.common.types
+
+import com.google.protobuf.Timestamp
+import de.hpi.cloud.common.Context
+import de.hpi.cloud.common.Persistable
+import de.hpi.cloud.common.protobuf.build
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.SerialClassDescImpl
+import java.time.ZoneOffset
+import java.time.LocalDateTime as RawLocalDateTime
+
+@Serializable(with = LocalDateTime.JsonSerializer::class)
+data class LocalDateTime(val value: RawLocalDateTime) : Persistable<LocalDateTime>() {
```

Comment:
i have the feeling that this is used like a timestamp - therefore https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html would be the correct class instead of LocalDateTime